body.tex

```diff
@@ -32,11 +51,21 @@ \subsection{Software Product} \label{sec:swprod}
 Where a SW Product is comprised of multiple git packages
 a Github \textit{metapackage} is used to identify the SW Product.
 All git packages of the SW product will be dependencies in a Github \textit{metapackage} and released at the same time.
-
-Note that git packages used to build the software product, should not be part of it.
+A git package shall be related only to one Software Product.
 
 Software products \textit{live} in a software repository. The LSST software repository is Github\footnote{\url{https://github.com/lsst}}.
 
+The following elements shall never be part of a Software Product:
```

Comment:
is suppose this is ok but i would phrase as "should not be part of "

Reply:
I rephrase it as you suggest. But keep in mind that this is the source of all DM problems.